app/src/main/java/com/ddoczi/tasky/agenda/domain/model/AgendaItem.kt

```diff
@@ -0,0 +1,56 @@
+package com.ddoczi.tasky.agenda.domain.model
+
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+sealed class AgendaItem(
+    val agendaItemId: String,
+    val title: String,
+    val description: String,
+    val time: LocalDate,
```

Comment:
An agenda item not only has a date, but also a time, so could it make sense to make this a `LocalDateTime`?